src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -190,13 +197,19 @@ impl<F: TableBuilderFactory> CapacitySplitTableBuilder<F> {
                 }
                 let ret = sstable_store.put_sst(sst_id, meta, data, policy).await;
                 drop(tracker);
+                if let Some(progress_tracker) = task_progress {
+                    progress_tracker.inc_ssts_uploaded();
+                }
                 ret
             });
             self.sealed_builders.push(SealedSstableBuilder {
                 sst_info,
                 upload_join_handle,
                 bloom_filter_size,
-            })
+            });
+            if let Some(progress_tracker) = &self.task_progress {
```

Comment:
I'm a little concerned about lock contention, even though we will only operate once per sst.

Is there a test here？ like benchmark or mirco_bench (like `bench_compactor`)

Reply:
Actually, there is potential contention across splits.

I have not done any bench or test so far.

Reply:
anw, I don't think it will be so bad. block size is 128MB meaning that one unit of execution will ask for lock relatively infrequently. While it may cause context switch of underlying threads if blocked on the lock, the underlying operation is very fast (it is just an increment of an integer) so I think the chance of that is low.

Reply:
I've changed it to increment a block counter instead of sst counter now, this makes things worse. In fact, to avoid lock contention, I believe it is sufficient to simply increment a counter for every SST produced as before. Anw, we do not have a very low latency requirement for the identification of stalled tasks and we currently already choose a 60 second task heartbeat timeout.

Reply:
Actually, now that I think about it, implementing as atomic may be more efficient. I'll create an issue for this.